src/format/opamFilter.mli

```diff
@@ -129,6 +129,8 @@ val ident_string: ?default:string -> env -> fident -> string
 (** Like [ident_value], but casts the result to a bool *)
 val ident_bool: ?default:bool -> env -> fident -> bool
 
+val expand_interpolations_in_file_full: env -> src:filename -> dst:filename -> unit
```

Comment:
```suggestion
val expand_interpolations_in_file_full: env -> src:filename -> dst:filename -> unit
(** Same as [expand_interpolations_in_file] but allows to set the source [src] and
    destination [dst] files independently instead of implying [src] = [dst].in *)
```